flink-connectors/flink-connector-kafka-0.10/src/test/java/org/apache/flink/streaming/connectors/kafka/Kafka010FetcherTest.java

```diff
@@ -119,7 +119,7 @@ public Void answer(InvocationOnMock invocation) {
         @SuppressWarnings("unchecked")
         SourceContext<String> sourceContext = mock(SourceContext.class);
         List<KafkaTopicPartition> topics = Collections.singletonList(new KafkaTopicPartition("test", 42));
-        KeyedDeserializationSchema<String> schema = new KeyedDeserializationSchemaWrapper<>(new SimpleStringSchema());
+        RichKeyedDeserializationSchemaWrapper<String> schema = new RichKeyedDeserializationSchemaWrapper<>(new SimpleStringSchema());
```

Comment:
This file will have conflict with the current `master`, because I recently pushed a hotfix to `master` to fix the indentation of this file (previously, it's incorrectly using spaces to indent instead of tabs). Sorry about this!